app/controllers/guest_rooms_controller.rb

```diff
@@ -0,0 +1,9 @@
+class GuestRoomsController < ApplicationController
+
+
+  # def create
```

Comment:
You were on the right track here! Doing this implementation here rather than in "guests#create" is a more RESTful option because we would ideally like to reserve "guests#create" for creating new Guest resources, rather than creating new joins resources. Your implementation here looks correct to me but let me know if you got stuck on something here that led you to put it in the GuestsController